Parsers/ASimNetworkSession/Tests/SentinelOne_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,7 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 11382 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SentinelOne""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 11382 records (100.0%) for field [EventVendor] of type [Enumerated]: [""SentinelOne""] (Schema:NetworkSession)"
+"(2) Info: Empty value in 10986 records (96.52%)  in optional field [SrcUsername] (Schema:NetworkSession)"
```

Comment:
SrcIpAddr or DvcIpAddr is all null in sample data:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/aefb5d21-b8b8-41c0-b2a2-82fc88488cd3)

Are we not getting this info much?

Reply:
Data is already available in raw as well as in csv file with the field name "alertInfo_srcIp" which was mapped with the SrcIpAddr and DvcIpAddr in the Parser.
<img width="807" alt="image" src="https://github.com/Azure/Azure-Sentinel/assets/98145046/90f5d6c8-da90-4aa9-b264-bb1e42194b81">